Task2/homework2/src/Homework2.java

```diff
@@ -0,0 +1,55 @@
+public class Homework2 {
+    public static void main(String[] args) {
+        System.out.println(checkSum(10, 20));
+        checkNumber(-10);
+        System.out.println(isPositive(24));
+        printStringNTimes("hello", 5);
+        System.out.println(isLeap(2020));
+    }
+
+//    Задание 1
+    public static boolean checkSum(int a, int b){
+        int s = a + b;
+        if (s >= 10 && s <= 20)
```

Comment:
поскольку логическое выражение само по себе возвращает нам boolean значение
аналогично и для других методов, то можно упростить
```suggestion
       return s >= 10 && s <= 20;
```